Add MMM ROAS Priors Case Study

juanitorduz opened the pull request:
Related to https://github.com/pymc-labs/pymc-marketing/issues/905

I adapt my case study https://juanitorduz.github.io/mmm_roas/  where I implement the model from the paper [“Media Mix Model Calibration With Bayesian Priors”, by Zhang, et al.](https://research.google/pubs/media-mix-model-calibration-with-bayesian-priors/), where the authors propose a convenient parametrization the regression model in terms of the ROAs (return on advertising spend) instead of the classical regression (beta) coefficients.

Here, we try to use the PyMC marketing lif-test approach to reproduce the results.

---

Also added `pyprojroot` following https://ericmjl.github.io/data-science-bootstrap-notes/use-pyprojroot-to-define-relative-paths-to-the-project-root/

<!-- readthedocs-preview pymc-marketing start -->
----
📚 Documentation preview 📚: https://pymc-marketing--916.org.readthedocs.build/en/916/

<!-- readthedocs-preview pymc-marketing end -->

juanitorduz commented on docs/source/notebooks/mmm/mmm_roas.ipynb:
aded!

juanitorduz commented:
@twiecki I worked on a more engaging intro. Please let me know what you think about it :)

twiecki commented:
Great stuff. One minor point of confusion, even though you say it in the intro, is that we don't model the confounder here, even though we might want to and that would be a great solution as well, just here we don't have it but can get around this through experimentation.

juanitorduz commented:
<p>done! thanks</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/916/discussion/#comment-2282889635'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

juanitorduz commented:
<p>+1</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/916/discussion/#comment-2282889637'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

juanitorduz commented:
<p>+1</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/916/discussion/#comment-2282889644'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

juanitorduz commented:
<p>XD!</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/916/discussion/#comment-2282889648'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

juanitorduz commented:
<p>+1</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/916/discussion/#comment-2282889650'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

juanitorduz commented:
<p>XD. I will do!</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/916/discussion/#comment-2282889655'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

juanitorduz commented:
Thank you, @twiecki, for your feedback! I addressed all the comments in https://github.com/pymc-labs/pymc-marketing/pull/916/commits/893ac51ab6d514d7753d1f758a41867b92f8aab5 . Apologies for so many typos. I thought I had fixed them before 🙈
